### DISCUSSION TITLE
Can't launch_duet using http://ec2-18-216-8-163.us-east-2.compute.amazonaws.com:5000/

### DISCUSSION THREAD

**Discussion by eslammofreh** [OpenMined/courses#447] [Course: Foundations of Private Computation]
**I get this error** _"ConnectionError: HTTPConnectionPool(host='ec2-18-216-8-163.us-east-2.compute.amazonaws.com', port=5000): Max retries exceeded with url: //metadata (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001B121531820>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"_
**when I execute**:
     _duet = sy.launch_duet(network_url="http://ec2-18-216-8-163.us-east-2.compute.amazonaws.com:5000/")_

**Comment by 910092** [comment]
you can try to check by keeping DUET server on data owner  itself instead from AWS server so duet = sy.launch_duet() could help .Please check not sure .Sorry for inconvenience if any.
